hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -821,25 +818,6 @@ public static boolean renameAndSetModifyTime(final FileSystem fs, final Path src
     return fs.rename(src, dest);
   }
 
-  /**
-   * Do our short circuit read setup.
-   * Checks buffer size to use and whether to do checksumming in hbase or hdfs.
-   * @param conf must not be null
-   */
-  public static void setupShortCircuitRead(final Configuration conf) {
```

Comment:
Not used?

Reply:
Doesn't seem to be the case. Removed it as `CommonFSUtils` is declared `InterfaceAudience.Private`.